joomla/joomla-cms
# Joomla Debug Console Improvements

#### Description

This PR adds some visual and informative improvements to the Joomla Debug Console.
##### Before PR (Database Queries -> "Call stack" slider)

![before-callstack](https://cloud.githubusercontent.com/assets/9630530/12310438/30545d98-ba48-11e5-8152-dc43da0f9249.png)
##### After PR (Database Queries -> "Call stack" slider)

![after-callstack](https://cloud.githubusercontent.com/assets/9630530/12310444/395cfba2-ba48-11e5-8f2a-c85199fcf3e3.png)
##### Before PR (Log Messages)

![before-logmessages](https://cloud.githubusercontent.com/assets/9630530/12310476/549ce5c6-ba48-11e5-863b-6864e553f19e.png)
##### After PR (Log Messages)

![after-logmessages](https://cloud.githubusercontent.com/assets/9630530/12310472/4feaf0b8-ba48-11e5-9355-e693453ebfcd.png)
#### How to test
1. Apply this patch.
2. Turn on the debug in global config and enable the debug system plugin.
3. Test any page in frontend or administrator and check the "Joomla Debug Console" in the bottom of the page.
4. Verify the info and visual of "Database Queries" -> "Call stack" slider
5. Verify the info and visual of "Log Messages"
#### Observations

Suggestions and improvements are welcome.
